src/ui/window/ShortcutManager.qml

```diff
@@ -69,8 +66,8 @@ Item {
     Shortcut {
         autoRepeat: false
         // it's a hack since StandardKey.PreviousChild not working for current Qt 5.7
-        // TODO: for mac it should be forced to Ctrl+Shift+Tab somehow
-        sequence: "Ctrl+Shift+Tab"
+        // TODO: for mac it should be forced to Ctrl+Shift+Tab
+        sequence: Qt.platform.os == "mac" ? "Meta+Shift+Tab" : "Ctrl+Shift+Tab"
```

Comment:
@JohnPoison according to [the docs](http://doc.qt.io/qt-5/qml-qtqml-qt.html) it's "osx", not "mac".